db/seeds/development/seedItems.js

```diff
@@ -1,15 +1,48 @@
 const faker = require('faker');
 
+var getRandomArbitrary = (min, max) => {
+  var number = Math.random() * (max - min) + min;
+  if(number > 0 && number < 0.75){
+    return 0.5
+   }
+  if(number >= 0.75 && number < 1.25){
+    return 1
+  }
+  if(number >= 1.25 && number < 1.75){
+    return 1.5
+  }
+  if(number >= 1.75 && number < 2.25){
+    return 2
+  }
+  if(number >= 2.25 && number < 2.75){
+    return 2.5
+  }
+  if(number >= 2.75 && number < 3.25){
+    return 3
+  }
+  if(number >= 3.25 && number < 3.75){
+    return 3.5
+  }
+  if(number >= 3.75 && number < 4.25){
+    return 4
+  }
+  if(number >= 4.25 && number < 4.75){
+    return 4.5
+  }
+  if(number >= 4.75 && number < 5){
+    return 5
+  }
+}
+
```

Comment:
This may not be relevant, since I am not sure what you need this rounding for --- but you can move something to a single decimal place using .toFixed() -- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
Thanks for the heads up on this Stacy, but don't think it will work on this because I want the random number to round up to the whole number or .5, this is for react-stars on the front-end because it only takes in integers at full or half a star.

Reply:
Ah yes, that's my mistake. Sorry about that! I see that you're incrementing on a 0.5 basis not just on a 0.1